File: FRS/FRS/ClientApp/src/app/models/meal-order/TrayModel.ts
export class TrayModel {
  constructor(id?: string) {
    this.id = id;
  }

  public id: string;
  public trayId: string;
  public plcId: string;
  public dispenserId: string;
  public ipAddress: string;
  public framework: string;

  public motorOutputNumber: number;
  public ledOutputNumber: number;
}

File: FRS/DAL/Models/MealOrder/TrayModel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DAL.Models.MealOrder
{
    public class TrayModel : AuditableEntity
    {
        [Key]
        public int Id { get; set; }

        public int DispenserId { get; set; }

        [ForeignKey(nameof(DispenserId))]
        public virtual DispenserOutlet DispenserOutlet { get; set; }

        public int PLCId { get; set; }

        [ForeignKey(nameof(PLCId))]
        public virtual PLCModel PLC { get; set; }

        public int MotorOutputNumber { get; set; }
        public int LEDOutputNumber { get; set; }
    }
}
